### DISCUSSION TITLE
Lumin OS (Magic Leap) for Unity 2020 versions not working "No text was generated from this prompt"

### DISCUSSION THREAD

**Discussion by SrProfessional** [srcnalt/OpenAI-Unity#58] [Q&A]
I have tried with Unity versions from 2020.3.0f1 to 2020.3.6f1 and  2020.3.48f1 and always appears the log "No text was generated from this prompt".

**Comment by srcnalt** [comment]
Hi @SrProfessional,

Could you check again with the latest release?

**Reply by SrProfessional** [reply]
Yes, was resolved as follows: Download the package manually from GitHub and import the downloaded folder as an asset in the Assets folder (Do not import it with the git URL in package manager).

**Reply by srcnalt** [reply]
Downloading with the git URL should work just fine. Did you encounter any issues with that?

**Reply by SrProfessional** [reply]
I'm going to try again with the last version by the git URL and comment if it works
